taipy/core/_scheduler/_scheduler_factory.py

```diff
@@ -9,17 +9,31 @@
 # an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
+from importlib import util
+
 from taipy.core._scheduler._abstract_scheduler import _AbstractScheduler
 from taipy.core._scheduler._scheduler import _Scheduler
 from taipy.core.common._utils import _load_fct
 from taipy.core.config.config import Config
 
 
 class _SchedulerFactory:
+
     @classmethod
     def _build_scheduler(cls) -> _AbstractScheduler:
+
         if Config.job_config._is_default_mode():
-            return _Scheduler()
+            if util.find_spec("taipy.enterprise"):
+                # Mode standalone with enterprise version
+                package = "taipy.enterprise.core.scheduler"
+                scheduler = _load_fct(package, "Scheduler")
+            else:
+                # Mode standalone with community version
+                scheduler = _Scheduler
+        else:
+            # Mode airflow with enterprise version
+            package = f"taipy.{Config.job_config.mode}.scheduler"
+            scheduler = _load_fct(package, "Scheduler")
```

Comment:
We should be consistent with https://github.com/Avaiga/taipy-core/blob/develop/taipy/core/config/job_config.py line 35.

Reply:
Maybe could we do it in an following card